docs/analyzer/user_guide.md

```diff
@@ -254,7 +254,8 @@ analyzer arguments:
   --analyzers ANALYZER [ANALYZER ...]
                         Run analysis only with the analyzers specified.
                         Currently supported analyzers are: clangsa, clang-
-                        tidy.
+                        tidy, cppcheck. Version number can be also specified 
```

Comment:
Can we give an example? Also, did we ever change the actual `--help` output? I can't see it anywhere.